client/src/components/inventory/HardwareItem.tsx

```diff
@@ -13,10 +13,34 @@ export interface Item {
     imageUrl: string
 }
 
+export interface RequestedItem {
+    id: number,
+    user: string,
+    name: string,
+    qtyRequested: number,
+    category: string,
+    status: ItemStatus,
+    cancelled: boolean
+}
+
+export enum ItemStatus {
+    SUBMITTED = "yellow",
+    APPROVED = "orange",
+    DECLINED = "red",
+    ABANDONED = "red",
+    READY = "blue",
+    FULFILLED = "green",
+    RETURNED = "grey",
+    LOST = "red",
+    DAMAGED = "red"
+}
+
 interface HardwareItem {
     qtyRemaining: number, // # of this item remaining in our stock
     requestsEnabled: boolean, // whether hardware requests can be made at this time
     toastManager: any, // for making toast notifications
+    addItem: (item: RequestedItem) => void,
+    qtyUpdate: RequestedItem | null,
```

Comment:
1. Took me a while to figure out what's going on with this (updating item quantity after an item request is cancelled), so please add a comment clarifying the purpose
2. For future reference, a way to selectively refresh the quantity remaining for items (or just websockets) would be a better way to do this.  But I appreciate that you took the time to think of this in the first place